src/components/Checkbox/tests/Checkbox.test.tsx

```diff
@@ -23,6 +24,16 @@ describe('<Checkbox />', () => {
       element.find('input').simulate('change');
       expect(spy).toHaveBeenCalledWith(true, 'MyCheckbox');
     });
+
+    it('renders with the correct focus when checkbox is toggled off', () => {
+      const input = mountWithAppProvider(
+        <Checkbox checked id="checkboxId" label="Checkbox" onChange={noop} />,
+      ).find('input');
+      (input.getDOMNode() as HTMLInputElement).checked = false;
+      input.simulate('change');
```

Comment:
Could you trigger the `onChange` instead?

Reply:
`trigger` will use a `keypath` to call to call functions on props. However the focus logic isn't user provided. It's contained in our `handleFocus` function, so we have to simulate a real change event.